Content.Server/GameObjects/Components/Disposal/DisposalUnitComponent.cs

```diff
@@ -483,10 +485,10 @@ protected override void Startup()
         {
             base.Startup();
 
-            Owner.EnsureComponent<AnchorableComponent>();
-
-            var collidable = Owner.EnsureComponent<CollidableComponent>();
-            collidable.AnchoredChanged += UpdateVisualState;
+            if (Owner.TryGetComponent(out CollidableComponent? collidable))
```

Comment:
A log warning should be displayed if the disposal unit doesn't have an anchorable component, as it's critical for it to function correctly.